integration_tests/integration_dotnet_test.go

```diff
@@ -535,42 +545,127 @@ func TestDeletedWith(t *testing.T) {
 func TestProviderCall(t *testing.T) {
```

Comment:
Looks like some stuff here wasn't properly gofmt'd, so this diff is unfortunately a bit noisier than it should be.